web/vtadmin/src/proto/vtadmin.d.ts

```diff
@@ -43087,6 +43135,12 @@ export namespace binlogdata {
 
         /** VEvent last_p_k_event */
         last_p_k_event?: (binlogdata.ILastPKEvent|null);
+
+        /** VEvent keyspace */
+        keyspace?: (string|null);
+
+        /** VEvent shard */
+        shard?: (string|null);
```

Comment:
@doeg @ajm188 these fields were added to the proto definition some time ago, but the vtadmin files don't seem to have been regenerated at that time.

Reply:
Can one of you post reminders in #maintainers and #developers so that people don't miss the step of generating vtadmin files?

Reply:
I've posted another reminder to both channels. (This was also mentioned back at the beginning of December.) I expect this will (understandably) be an issue until we can enforce this in CI. The ticket for that is https://github.com/vitessio/vitess/issues/9571.